dags/sagemaker-pipeline.py

```diff
@@ -142,23 +70,87 @@ def data_prep(data_url, s3_bucket, input_s3_key):
         s3_hook.load_file('iris_train.csv', '{0}/train.csv'.format(input_s3_key), bucket_name=s3_bucket, replace=True)
         s3_hook.load_file('iris_test.csv', '{0}/test.csv'.format(input_s3_key), bucket_name=s3_bucket, replace=True)
 
-    data_prep = data_prep(data_url, s3_bucket, input_s3_key)
+
+    data_prep = data_prep(data_url, "{{ var.value.get('s3_bucket') }}", input_s3_key)
```

Comment:
> You could also set it to a variable in the DAG too since it's used in several places.

@josh-fell It was originally this way, but I moved away from this so that users know which variables can stay the same and which ones have to change to get the DAG running. Basically add the Airflow Variables and you are good to go. I can change it back to that if its less confusing, but this way they don't really have to edit the Dag files.

Reply:
Meh, no strong opinion really.